### PR TITLE
Update Help window for WorkBook

### PR DESCRIPTION
**Pull request by glemenneo**
## Overview
1. Update User Guide link from AB3 to WorkBook website
2. Added formatting examples for all commands currently available in WorkBook
3. Added headers for commands briefly explaining their function
4. Styling changes such as colouring and bolding for text headers.
5. CSS rulesets have also been grouped together for the respective element types like headers and containers for easy changes in the future
---
Help window has been updated to reflect the functionality of WorkBook and can now be used as a quick reference for commands instead of just linking to the User Guide.

As more commands and functionalites are added to WorkBook, the Help window will need to be updated as well.

UI styling has not been finalised and this is just a preliminary version for the WorkBook v1.2 MVP.

### CODE REVIEW

File: src/main/resources/view/HelpWindow.fxml [hunk 1]
```diff
@@ -7,9 +7,11 @@
 <?import javafx.scene.control.Label?>
 <?import javafx.scene.image.Image?>
 <?import javafx.scene.layout.HBox?>
+<?import javafx.scene.layout.VBox?>
+<?import javafx.scene.text.Font?>
 <?import javafx.stage.Stage?>
 
-<fx:root resizable="false" title="Help" type="javafx.stage.Stage" xmlns="http://javafx.com/javafx/11" xmlns:fx="http://javafx.com/fxml/1">
+<fx:root resizable="false" title="Help" type="javafx.stage.Stage" xmlns="http://javafx.com/javafx/18" xmlns:fx="http://javafx.com/fxml/1">
```
Comment (rylzxc) [inline src/main/resources/view/HelpWindow.fxml:14]: I suggest to stick to javafx11 as it's the Long Term Support (LTS) version. Will it affect your current changes if you switched it back?

File: src/main/resources/view/HelpWindow.css [hunk 2]
```diff
@@ -1,4 +1,4 @@
-#copyButton, #helpMessage {
+#copyButton, #urlMessage, #addCommand, #clearCommand, #deleteCommand, #editCommand, #exitCommand {
```
Comment (jacobkwan) [inline src/main/resources/view/HelpWindow.css:1]: As discussed earlier, could experiment with using `styleClass` and css classes to achieve DRY :)

Comment (glemenneo) [reply src/main/resources/view/HelpWindow.css]: I've grouped the relevant elements together into styleClasses like headers and container, thanks for the tip!

**Review by jacobkwan — APPROVED** [review]
LGTM

**Pull request merged**